helix-view/src/view.rs

```diff
@@ -118,9 +118,6 @@ impl View {
                 width as usize,
             ));
         }
-        if !gutter_types.is_empty() {
-            gutter_offset += 1;
-        }
```

Comment:
This is a margin that's meant to separate the gutters and the code, themed as background

See #3111